machine/netberg/netberg_common_rangeley/INSTALL

```diff
@@ -0,0 +1,224 @@
+Add support for Netberg Common Rangeley CPU board
+
+================================================================================
+  Netberg Common Rangeley CPU board
+================================================================================
+
+  Ingrasys switch which supports modularized CPU board design, have a
+non-Netberg H/W component for mother board detection. For motherboard
+detection, please refer to ``busybox/miscutils/mb_detect.c``
+
+  Supported Platforms
+=======================
+- Aurora 620  (cpu module)
+   48x25GbE SFP28 ports and 6x100GbE QSFP28 ports
+
+- Aurora 720  (cpu module)
+   32x100GbE QSFP28 ports
+
+  Specification
+===============
+ - Modularized CPU board design
+ - 1 RJ-45 out-of-band management port (10/100/1000M)
+ - 1 console port (RJ-45 on Aurora 720, Mini-USB on Aurora 620)
+ - 1 8GB DDR3 SDRAM
+ - 1 SSD 32GB
+ - 1 USB 2.0 port
+
```

Comment:
Some of this detail about the CPU module should also go into the commit message.  It is very handy to find this info in the `git log`.